### DISCUSSION TITLE
The 1.6 PRE-RELEASE of Kade Engine

### DISCUSSION THREAD

**Discussion by Kade-github** [KadeArchive/Kade-Engine#1367] [General]
Additions:
- Full reworked charter
- Note quantization
- BPM Changes & Scroll Speed Change Support
- StepMania File Format support
- Anti-Aliasing option for low end pc's
- Pre-loading for characters option
- New Logo
- New Main Menu Remix
- Losing Icon for Father Dearest & Mommy Mearest together
- Difficulty Calculator
- Week Locking
- Character Offset Files

Fixes:
- BF Animation ending too early
- Held notes being weird as fuck (completely reworked them)
- Held note end fix (this made me mad)
- Up arrow having a 8 pixels next to it
- Stop crashes with "R"
- Reworked Health Drain to be harsher
- Fixed dropped inputs for stacked notes.
- Many beat based things actually go to the beat (blue balls, title screen, etc)
- Fixed multiple crashes
- Fixed spirit being weird
- Fixed a lot of desyncs

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.6-PRE-RELEASE'>The 1.6 PRE-RELEASE of Kade Engine</a>.</em>

**Comment by BenicioGPXD** [comment]
i love kadedev

**Comment by guizinhomartinez** [comment]
tbh i hated the week lock thing because now you need to end the entire game just to see if a dialogue or a cutcene is working or not
there should be a option to disable that

**Reply by Spel0** [reply]
Then compile a debug build... it has everything unlocked from the start

**Reply by guizinhomartinez** [reply]
oh, ok

**Reply by jukeliv** [reply]
you know than you can just, change the code and unlock everything, right?
